frontend/src/components/AdminSettings.vue

```diff
@@ -226,7 +226,7 @@ const betaUpdateExists = computed(() => {
 });
 
 onMounted(async () => {
-  keycloakURL.value = config.get().keycloakUrl;
+  keycloakAdminRealmURL.value = config.get().keycloakAdminRealmUrl;
```

Comment:
Since we already have both the Keycloak base url as well as the realm id, I believe we can do the concatenation part in javascript, keeping the DTO small.

```suggestion
  let cfg = config.get();
  keycloakAdminRealmURL.value = `${cfg.keycloakUrl}/admin/${cfg.keycloakRealm}/console`;
```

Reply:
I also thought about it but came to the conclusion that it is rather the task of the backend to provide such URLs but if you want, I'll change it.

Reply:
I think what I dislike about it coming from the backend is that it suggests this url could be configured separately. When in reality it is not really a standalone configuration but rather a derived property.